### ISSUE TITLE
Adding skills

### ISSUE THREAD

**Issue by rndmsfree** [sammachin/AlexaPi#27]
Never mind.  Figured it out, for what it's worth.

**Comment by maso27** [comment]
The shortcoming of AlexaPi is that it's currently only a state machine that waits for a button press, then records a request and returns a response. Anything that requires server-initiated audio beyond that one response gets dropped on the floor. 
There's still a large amount of functionality that does work under those constraints.

I've installed a number of skills on mine, and most work.  I've had success with The Bartender, Teacher, Spelling Bee, and The Magic Door.

**Comment by doubledrat** [comment]
how do you go about adding skills to AlexaPi?  I can't connect the amazon app to mine...

**Comment by maso27** [comment]
For me, it was fairly seamless.

My Android phone is on the same network as the AlexaPi, and I installed the Alexa app on the phone.
When I went into "Settings/Alexa Devices" I saw both my Fire TV stick and the RaspberryEcho.  For some reason the RaspberryEcho is listed as Offline (most likely due to the state machine that just sits idle instead of constantly communicating with the server).  I set my address, connected calendars, and all the other setup stuff in spite of the Offline status, and everything works.

I can also connect to it from my wife's iPhone, FWIW.
